jenkins/internal-ci/centos-7.9.2009/pr/kubernetes/motr-k8-cicd.groovy

```diff
@@ -0,0 +1,335 @@
+pipeline {
+	agent {
+		node {
+			label "docker-centos-7.9.2009-node"
+		}
+	}
+	options {
+		timeout(time: 180, unit: 'MINUTES')
+		timestamps()
+		skipDefaultCheckout()
+	}
+	parameters {
+		string(name: 'MOTR_URL', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Repo for Motr')
+		string(name: 'MOTR_BRANCH', defaultValue: 'kubernetes', description: 'Branch for Motr')
+		choice (
+			choices: ['DEBUG', 'ALL', 'DEVOPS'],
+			description: 'Email Notification Recipients ',
+			name: 'EMAIL_RECIPIENTS'
+		)
+	}
+	environment {
+		OS_VERSION = "centos-7.9.2009"
+		CORTX_RE_BRANCH = "kubernetes"
+		CORTX_RE_REPO = "https://github.com/Seagate/cortx-re/"
+
+		GPR_REPO = "https://github.com/${ghprbGhRepository}"
+		MOTR_URL = "${ghprbGhRepository != null ? GPR_REPO : MOTR_URL}"
+		MOTR_BRANCH = "${sha1 != null ? sha1 : MOTR_BRANCH}"
+		MOTR_GPR_REFSEPEC = "+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*"
+		MOTR_BRANCH_REFSEPEC = "+refs/heads/*:refs/remotes/origin/*"
+		MOTR_PR_REFSEPEC = "${ghprbPullId != null ? MOTR_GPR_REFSEPEC : MOTR_BRANCH_REFSEPEC}"
+		
+		//////////////////////////////// BUILD VARS //////////////////////////////////////////////////
+		// hosts,SNS_CONFIG, DIX_CONFIG, CORTX_SCRIPTS_BRANCH, CORTX_SCRIPTS_REPO, DOCKER_REGISTRY, PUSH_IMAGE and TAG_LATEST are manually created parameters in jenkins job.
```

Comment:
No need to add below parameters manually.
`SNS_CONFIG, DIX_CONFIG, CORTX_SCRIPTS_BRANCH, CORTX_SCRIPTS_REPO, DOCKER_REGISTRY, PUSH_IMAGE and TAG_LATEST`
we can pass them directly with values as below,
![image](https://user-images.githubusercontent.com/61046500/147544339-9ed19c3a-c8f4-44ca-b091-34dcb8f9bd8f.png)

Reply:
For single-node SNS should be 1+0+0 and for 3 node it should be 2+1+0